packages/pluggableWidgets/video-player-native/src/VideoPlayer.tsx

```diff
@@ -36,27 +53,129 @@ export function VideoPlayer(props: VideoPlayerProps<VideoStyle>): ReactElement {
         setStyles(alteredStyles);
     }, [props.style, props.aspectRatio, videoAspectRatio]);
 
+    useEffect(() => {
+        if (props.showControls) {
+            showControlsHandler();
+        }
+    }, [fullScreen, props.showControls]);
```

Comment:
`fullScreen` probably shouldn't be in the dependency list

Reply:
Adding `showControlsHandler` once you apply `useCallback` should

Reply:
The intended logic here is to call `showControlsHandler()` when fullscreen is toggled, how can we achieve that with using `useCallback`?

Reply:
Afte discussing, `fullScreen` makes sense for this effect